docs/sphinx_api_docs_source/public_api_report.py

```diff
@@ -1843,7 +1843,7 @@ def main():
     # any methods or classes you are adding to documentation with the @public_api
     # decorator and any relevant "new" or "deprecated" public api decorators.
     # If the actual is lower than the threshold, please reduce the threshold.
-    PUBLIC_API_MISSING_THRESHOLD = 97  # TODO: reduce this number again once this works for the Fluent DS dynamic methods
+    PUBLIC_API_MISSING_THRESHOLD = 99  # TODO: reduce this number again once this works for the Fluent DS dynamic methods
```

Comment:
I cranked this up by 2 since this has to do with fluent ds dynamic methods which the comment highlights as not working correctly at the moment.